pkg/parse/parse.go

```diff
@@ -480,6 +480,12 @@ func PullPolicyFromFlagSet(flags *pflag.FlagSet, findFlagFunc func(name string)
 	if strings.EqualFold(pullFlagValue, "true") || strings.EqualFold(pullFlagValue, "ifnewer") {
 		pullPolicy = define.PullIfNewer
 	}
+	if findFlagFunc("pull").Changed {
+		if strings.EqualFold(pullFlagValue, "false") {
+			pullPolicy = define.PullNever
+		}
+	}
+
```

Comment:
Maybe make this another value that the block that currently checks for "never" accepts?